src/xenia/gpu/vulkan/texture_cache.cc

```diff
@@ -325,17 +350,21 @@ bool TextureCache::FreeTexture(Texture* texture) {
     vkDestroyFramebuffer(*device_, texture->framebuffer, nullptr);
   }
 
-  for (auto it = texture->views.begin(); it != texture->views.end();) {
-    vkDestroyImageView(*device_, (*it)->view, nullptr);
-    it = texture->views.erase(it);
+  for (auto region_it = texture->regions.begin();
```

Comment:
You don't appear to be freeing the base_region here

Reply:
Also - there's a regression somewhere causing a texture to be freed twice in this branch, but I haven't tracked it down quite yet.